src/utils/json.ts

```diff
@@ -1,6 +1,5 @@
 import { display as displayDataCell } from '../ui/RealmBrowser/Content/Table/types/DataCell';
-import { stringify } from 'flatted';
-
+import { InspectOptions, inspect } from 'util';
```

Comment:
```suggestion
import { InspectOptions, inspect } from 'node:util';
```

Reply:
Is there an advantage to this import over the other, is it to make it explicit this is from part of node? We actually also import `util` directly in an earlier file (which I chose to leave untouched to avoid unnecessary side effects) so I could also change it there
https://github.com/realm/realm-studio/blob/48b64e14733c8848db191436cdb9e8ea76bf2fda/src/ui/RealmBrowser/display.ts#L19